src/css_to_emotion.re

```diff
@@ -1126,33 +1125,98 @@ and render_declaration_list = ((list, loc): Declaration_list.t, variables): expr
 }
 and render_style_rule = (ident, sr: Style_rule.t): expression => {
   let (prelude, prelude_loc) = sr.Style_rule.prelude;
-  let selector =
-    List.fold_left(
-      (s, (value, value_loc)) =>
-        switch (value) {
-        | Delim(":") => ":" ++ s
-        | Ident(v)
-        | Operator(v)
-        | Delim(v)
-        | Selector(v) =>
-          if (String.length(s) > 0) {
-            v ++ " " ++ s;
-          } else {
-            v ++ s;
-          }
-        | _ => grammar_error(value_loc, "Unexpected selector")
-        },
-      "",
-      List.rev(prelude),
-    );
-  let selector_expr = string_to_const(~loc=prelude_loc, selector);
   let dl_expr = render_declaration_list(sr.Style_rule.block, None);
-
-  Exp.apply(
-    ~loc=sr.Style_rule.loc,
-    ident,
-    [(Nolabel, selector_expr), (Nolabel, dl_expr)],
-  );
+  switch (prelude) {
+  | [(Selector("&"),_),(Delim(":"),_),(Delim(":"),_),(Ident(pc),loc)] =>
+    let f = switch (pc) {
+    | "active" => "active"
+    | "after" => "after"
+    | "before" => "before"
+    | "first-line" => "firstLine"
+    | "first-letter" => "firstLetter"
+    | "selection" => "selection"
+    | "placeholder" => "placeholder"
+    | _ => grammar_error(loc, "Unexpected pseudo-class")
+    };
+    let ident = Exp.ident(~loc, {txt: Emotion.lident(f), loc});
+    Exp.apply(
+      ~loc=sr.Style_rule.loc,
+      ident,
+      [(Nolabel, dl_expr)],
+    );
+  | [(Selector("&"),_),(Delim(":"),_),(Ident(pc),loc)] =>
+    let f = switch (pc) {
+    | "checked" => "checked"
+    | "disabled" => "disabled"
+    | "first-child" => "firstChild"
+    | "first-of-type" => "firstOfType"
+    | "focus" => "focus"
+    | "hover" => "hover"
+    | "last-child" => "lastChild"
+    | "last-of-type" => "lastOfType"
+    | "link" => "link"
+    | "read-only" => "readOnly"
+    | "required" => "required"
+    | "visited" => "visited"
+    | "enabled" => "enabled"
+    | "empty" => "noContent"
+    | "default" => "default"
+    | "any-link" => "anyLink"
+    | "only-child" => "onlyChild"
+    | "only-of-type" => "onlyOfType"
+    | "optional" => "optional"
+    | "invalid" => "invalid"
+    | "out-of-range" => "outOfRange"
+    | "target" => "target"
+    | _ => grammar_error(loc, "Unexpected pseudo-class")
```

Comment:
nth-child doesn't appear here, but appears on the test. am I missing something here?